client/layout/controller.js

```diff
@@ -109,14 +109,14 @@ export class Controller extends Component {
 
 	getBaseQuery( searchString ) {
 		const query = this.getQuery( searchString );
-		delete query.page;
+		delete query.paged;
 		return query;
 	}
 
 	render() {
 		const { page, match, location } = this.props;
 		const { url, params } = match;
-		const query = this.getBaseQuery( location.search );
+		const query = this.getQuery( location.search );
```

Comment:
The point of `getBaseQuery` is to remove the unused parameters. In the case of the updated url scheme, we want to pass down everything except for `page: "wc-admin"`, because its irrelevant to the app. `paged` however is important and should be passed down.

So `getBaseQuery` should remove `page` parameter, and this line should go back to using `getBaseQuery`.

Reply:
Ah I see. I think there will need to be changes to `componentDidUpdate` as well, since it compares base queries to determine if it needs to reset the page number. If `paged` is part of the base query, I believe it won't allow getting to page 2+.

Reply:
Fixed in 4ff4b32.

Reply:
>If paged is part of the base query, I believe it won't allow getting to page 2+.

Indeed thats the case. I did some testing by removing `delete query.paged;` and it could not get to page 3. 

`getBaseQuery` seems like a utility function to be used in a variety of places. Can we rename it to `getPagedLessQuery`? Or something like that?

Reply:
I only see `getBaseQuery` used within the controller file - where else are you seeing it?

Reply:
Nowhere else, I'm just hoping to prevent future confusion

Reply:
What do you think about just omitting the `page` parameter in `getQuery()` and removing `getBaseQuery()` entirely? Seems like we never want to use the `page` value since it's going to be the WordPress-registered page anyhow.

Reply:
>What do you think about just omitting the page parameter in getQuery()

That sounds ideal.

>and removing getBaseQuery() entirely?

Also idea, just remember to remove `paged` for the comparison made in `componentDidUpdate`

Reply:
Done in 697b8a7.

Reply:
>What do you think about just omitting the page parameter in getQuery()

Looking great, did this part get omitted?

Reply:
Yeah - I didn't initially realize it was unnecessary. It was to catch the now defunct `page` fragment param for pagination.